lib/ansible/modules/monitoring/zabbix/zabbix_service.py

```diff
@@ -129,42 +153,39 @@ def delete_service(self, service_ids):
         self._zapi.service.delete(service_ids)
 
     def dump_services(self, service_ids):
-        services = self._zapi.service.get({'output': 'extend', 'filter' : {'serviceid' : service_ids}})
+        services = self._zapi.service.get({'output': 'extend', 'filter': {'serviceid': service_ids}, 'selectParent': '1'})
         return services
 
     def generate_service_config(self, name, parent, sla, calculate_sla, trigger_name, trigger_host, algorithm):
```

Comment:
def generate_service_config(self, name, parent, sla, calculate_sla=0, trigger_name, trigger_host, algorithm):

This simplify a little the code later